automata/fa/nfa.py

```diff
@@ -7,6 +7,7 @@
 from pydot import Dot, Edge, Node
 
 import automata.base.exceptions as exceptions
+from automata.parse.parser import parse_regex
```

Comment:
@eliotwrobson Is the `automata.parse.parser` module specific to parsing only regexes? If so, I think a less-ambiguous name like `automata.regex.parse` would be better.

If we do this, we should also move `automata.base.regex` to `automata.regex.regex`, so that everything regex-related is under the same sub-package.

Reply:
Only the `automata.parse.parser` module is specific to regexes, the `lexer` and `postfix` modules are actually completely generic. But I think you're right, it might be best to put this all in a regex sub-package (even if the other components are generic).

Reply:
@eliotwrobson Okay, great! Please proceed with that reorg, when you can.